userspace/engine/falco_engine.cpp

```diff
@@ -86,6 +86,28 @@ uint32_t falco_engine::engine_version()
 
 #define CONSOLE_LINE_LEN 79
 
+static void wrap_text(const std::string &str, uint32_t initial_pos, uint32_t indent, uint32_t line_len)
```

Comment:
This function should only have the responsibility to construct the string which then can be printed.

Furthermore I'll move this in an outside file (eg., utils, with `falco::engine::utils` namespace maybe). WDYT

Reply:
Done.